content/md/products-super-power/enrich-products-variants.md

```diff
@@ -72,9 +85,13 @@ For more details about the product form, you can read [Enrich your product](/art
 
 ## Header and infos
 
-For products with variants, the number of variant products behind the model and the number of complete variant products for the model are displayed in the product form header (`Variant products`).
+For product models, the following properties are displayed in the product form header:
+- The number of complete variant products out of the number of variant products for a product model in `Variant products` (ex: 2/6 means that 2 variants out of 6 for the product model are complete)
+- The total `missing required attributes` (__since the 2.1__) for a product model. If you click on it, it will display only the attributes required for the completeness without value (filter on display `all missing required attributes`).
```

Comment:
`the attribute that are required for the completeness but that are still incomplete`

Reply:
Also, I don't understand what you wanna say by `(filter on display `all missing required attributes`)`

Reply:
ok